services/movie_session.py

```diff
@@ -42,3 +42,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    return list(
+        MovieSession.objects.get(
+            pk=movie_session_id).tickets.all().values("row", "seat")
```

Comment:
In this case, you do 2 queries. Change code to use only one query(Use Ticket as base model)

Reply:
Thx you. Changed.